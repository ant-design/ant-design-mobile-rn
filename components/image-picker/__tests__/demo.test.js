import rnDemoTest from '../../../tests/shared/demoTest'

rnDemoTest('image-picker')
