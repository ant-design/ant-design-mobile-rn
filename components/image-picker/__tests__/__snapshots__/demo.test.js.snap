// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/image-picker/demo/basic.tsx correctly 1`] = `
<View
  style={
    Object {
      "marginLeft": 20,
      "marginTop": 20,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "flexWrap": "wrap",
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <View
      style={
        Array [
          Object {
            "marginBottom": 6,
            "marginRight": 5,
            "overflow": "hidden",
          },
          Object {
            "height": 80,
            "width": 80,
          },
        ]
      }
    >
      <
        activeOpacity={0.6}
        onPress={[Function]}
      >
        <Image
          source={
            Object {
              "uri": "https://zos.alipayobjects.com/rmsportal/WCxfiPKoDDHwLBM.png",
            }
          }
          style={
            Array [
              Object {
                "height": 80,
                "width": 80,
              },
              Object {
                "borderRadius": 3,
                "overflow": "hidden",
              },
            ]
          }
        />
      </>
      <
        activeOpacity={0.6}
        onPress={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#999",
            "borderRadius": 8,
            "height": 16,
            "justifyContent": "center",
            "overflow": "hidden",
            "position": "absolute",
            "right": 4,
            "top": 4,
            "width": 16,
          }
        }
      >
        <Text
          style={
            Object {
              "backgroundColor": "transparent",
              "color": "#ffffff",
              "fontSize": 20,
              "fontWeight": "300",
              "height": 20,
              "marginTop": -8,
            }
          }
        >
          ×
        </Text>
      </>
    </View>
    <TouchableWithoutFeedback
      onPress={[Function]}
      onPressIn={[Function]}
      onPressOut={[Function]}
    >
      <View
        style={
          Array [
            Object {
              "marginBottom": 6,
              "marginRight": 5,
              "overflow": "hidden",
            },
            Object {
              "height": 80,
              "width": 80,
            },
            Object {
              "alignItems": "center",
              "borderRadius": 3,
              "borderWidth": 1,
              "justifyContent": "center",
            },
            Object {
              "backgroundColor": "#ffffff",
              "borderColor": "#dddddd",
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "color": "#888888",
                "fontSize": 64,
                "fontWeight": "100",
              },
            ]
          }
        >
          +
        </Text>
      </View>
    </TouchableWithoutFeedback>
  </View>
  <View
    style={
      Array [
        Object {
          "height": 9,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "flexDirection": "row",
        "flexWrap": "wrap",
      }
    }
  >
    <TouchableWithoutFeedback
      onPress={[Function]}
      onPressIn={[Function]}
      onPressOut={[Function]}
    >
      <View
        style={
          Array [
            Object {
              "marginBottom": 6,
              "marginRight": 5,
              "overflow": "hidden",
            },
            Object {
              "height": 80,
              "width": 80,
            },
            Object {
              "alignItems": "center",
              "borderRadius": 3,
              "borderWidth": 1,
              "justifyContent": "center",
            },
            Object {
              "backgroundColor": "#ffffff",
              "borderColor": "#dddddd",
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "color": "#888888",
                "fontSize": 64,
                "fontWeight": "100",
              },
            ]
          }
        >
          +
        </Text>
      </View>
    </TouchableWithoutFeedback>
  </View>
</View>
`;
